From 9d81aa4505d907528f5be5718264db7b1a1ee9ff Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Mon, 12 May 2025 22:41:01 +0300
Subject: [PATCH] [SW-227605] g2e: fix support G2E variant

In hl-smi, GAUDI2E (OAM) device type was added,
but GAUDI2E HL-288 (PCIe) device type was missed.

Change-Id: I2b745261b5eb408ba936ed901efbb5223e2765d5
Signed-off-by: Meron Cohen <mecohen@habana.ai>
---

diff --git a/src/utils.c b/src/utils.c
index da4a49e..14a8b70 100644
--- a/src/utils.c
+++ b/src/utils.c
@@ -744,6 +744,7 @@
 	"GAUDI3 HL-338",
 	"GAUDI2 HL-288",
 	"GAUDI2D HL-288",
+	"GAUDI2E HL-288"
 };
 
 static bool is_device_supported(const char *devname)
